spring-security/src/test/java/com/sap/cloud/security/spring/config/XsuaaServiceConfigurationsTest.java

```diff
@@ -5,30 +5,55 @@
  */
 package com.sap.cloud.security.spring.config;
 
+import com.sap.cloud.environment.servicebinding.SapVcapServicesServiceBindingAccessor;
+import com.sap.cloud.security.config.Service;
+import com.sap.cloud.security.config.ServiceBindingEnvironment;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 
+import java.io.IOException;
+import java.util.List;
+
+import static com.sap.cloud.security.spring.config.XsuaaServiceConfigurationTest.assertConfigsAreEqual;
```

Comment:
Qualms... in `src/test/main/` you may have util classes, which will host such "helper methods".